Firmware/mmu2.cpp

```diff
@@ -839,6 +837,13 @@ void MMU2::execute_extruder_sequence(const E_Step *sequence, uint8_t steps) {
     }
 }
 
+void MMU2::execute_load_to_nozzle_sequence() {
+    st_synchronize();
+    // Compensate for configurable Extra Loading Distance
+    current_position[E_AXIS] -= (logic.ExtraLoadDistance() - MMU2_FILAMENT_SENSOR_POSITION);
```

Comment:
/nitpick: if extra load distance is set to some insane value, this subtraction can cause the extruder to actually unload the filament :wink:

Reply:
Yeah, I know. When I picked the value for testing I had to make sure not to exceed that. I don't think it's necessary to add an extra check for that since only knowlageable users (or print profile makers) should use the register anyway.

Reply:
I agree, that register should be primarily set with a hard-coded per-printer-type value (or may be some mods may need to tweak this value). I don't see a reason why a user would want to tweak extra loading distance per G-code or even at runtime.